spec/cli_spec.rb

```diff
@@ -71,24 +72,77 @@
 
       before do
         allow(role).to receive(:create)
-        allow(ElasticWhenever::CLI).to receive(:clear_tasks).with(kind_of(ElasticWhenever::Option))
+
         allow_any_instance_of(ElasticWhenever::Task::Rule).to receive(:create)
         allow_any_instance_of(ElasticWhenever::Task::Target).to receive(:create)
+        allow(ElasticWhenever::Task::Rule).to receive(:fetch).and_return([])
       end
 
-      it "updates tasks and returns success status code" do
-        expect(role).to receive(:create)
```

Comment:
Creating a role is an important concern, so it might be a good idea to create a test for this.